rerun_py/rerun_sdk/rerun/__init__.py

```diff
@@ -214,28 +211,33 @@ def is_enabled() -> bool:
     return bindings.is_enabled()  # type: ignore[no-any-return]
 
 
-def set_enabled(enabled: bool) -> None:
-    """
-    Enable or disable logging.
+def rerun_shutdown() -> None:
+    bindings.shutdown()
 
-    If false, all calls to the rerun library are ignored. The default is `True`.
 
-    This is a global setting that affects all threads.
+atexit.register(rerun_shutdown)
 
-    By default logging is enabled, but can be controlled with the environment variable `RERUN`,
-    (e.g. `RERUN=on` or `RERUN=off`).
 
-    The default can be set in [`rerun.init`][].
+def unregister_shutdown() -> None:
+    atexit.unregister(rerun_shutdown)
 
-    Parameters
-    ----------
-    enabled : bool
-        Whether to enable or disable logging.
 
-    """
-    bindings.set_enabled(enabled)
+# ---
 
 
+# TODO
+def get_active_data_recording() -> RecordingStream:
+    """TODO."""
+    return RecordingStream(bindings.get_active_data_recording())
+
+
+# TODO
+def set_active_data_recording(recording: RecordingStream):
+    """TODO."""
+    return bindings.set_active_data_recording(recording.inner)
+
+
+# TODO: not sure what this becomes
```

Comment:
it doesn't change, this is actually our one and only purely statically global thingy